src/librbd/exclusive_lock/PostAcquireRequest.cc

```diff
@@ -17,6 +17,10 @@
 #include "librbd/image/RefreshRequest.h"
 #include "librbd/journal/Policy.h"
 
+#if defined(WITH_RBD_RWL)
```

Comment:
Nit: I'd prefer if all of this was hidden `InitRequest` to reduce the spread of `#if defined(WITH_RBD_RWL)`. Longer term we can get these hooks registered as part of the plug-in library hook points.

Reply:
Moved it to rwl/InitRequest.